data/$Reach/data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Reach",
+  "symbol": "$Reach",
+  "decimals": 18,
+  "description": "The /Reach platform fosters a collaborative environment where creators and contributors unite to enhance the visibility of Web3 projects, while also reaping rewards for their participation.",
+  "website": "https://getreach.xyz",
+  "twitter": "https://twitter.com/GetReachxyz",
+  "tokens": {
+    "ethereum": {
+      "address": "0x8b12bd54ca9b2311960057c8f3c88013e79316e3"
+    },
+    "base": {
+      "address": "0x4379c13143eb91148ff9282cfb2f93536687a45b"
```

Comment:
Is this a valid Base contract address?

Reply:
Yes it is : https://basescan.org/address/0x4379c13143Eb91148fF9282CFb2f93536687A45b#internaltx
it has been deployed using base-migrate.xyz